lib/chef-dk/policyfile_compiler.rb

```diff
@@ -79,7 +95,17 @@ def cookbook_location_spec_for(cookbook_name)
 
     def expanded_run_list
       # doesn't support roles yet...
-      Chef::RunList.new(*run_list)
+      concated_runlist = Chef::RunList.new
+      included_policies.each do |policy_spec|
+        lock = policy_spec.policyfile_lock
+        lock.run_list.each do |run_list_item|
+          concated_runlist << run_list_item
```

Comment:
Might want to leave a note here and below as to why this isn't `concat(lock.run_list)` or similar, otherwise looks kind of like a bug unless you know RunList only partially implements the Array API :)

Reply:
was either trial and error or i didn't know about concat :P

Reply:
Heh, this is correct I think, RunList doesn't actually have a `concat` :) Just would be incorrect if it were an Array so it set off my "possible bug" alarm while scanning.